### PR TITLE
Improve SEO page scan

### PR DESCRIPTION
**Pull request by ValJed**
* Scan internal and external links
* Scan images to check alt attributes
* Scan Open graph title and description
* Implement design

If you want to try the feature, you can simply link the module inside any apostrophe project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,9 +65,43 @@ If you would like to configure additional fields to allow an editor to add a Goo
...
+      },
+      ogTitleLength: {
+        min: 40,
+        max: 80
+      },
+      ogDescriptionLength: {
+        min: 40,
+        max: 80
+      },
+      minWordsNumber: 400
```
Comment (boutell) [inline README.md]: Let's make this `minWordsOnPage`

File: lib/api.js [hunk 2]
```diff
@@ -111,13 +160,30 @@ module.exports = (self, { scanRules }) => {
     if (totalWords < minWordsNumber) {
       return {
         error: true,
-        msg: `There should be at least ${minWordsNumber} words in the page,
-        there are currently ${totalWords}`
+        msg: `There are <strong>${totalWords}</strong> words on the page,
+        which does not meet the minimum requirements of <strong>${minWordsNumber}</strong>.`
```
Comment (boutell) [inline lib/api.js]: Minimum requirement, not requirements

File: lib/api.js [hunk 3]
```diff
@@ -111,13 +160,30 @@ module.exports = (self, { scanRules }) => {
...
       };
     }
     return {
       error: false,
-      msg: `There are enough content on this page: ${totalWords} words`
+      msg: `There are <strong>${totalWords}</strong> words on the page, which meets the minimum requirements of <strong>${minWordsNumber}</strong>.`
```
Comment (boutell) [inline lib/api.js]: requirement, not requirements

**Review by boutell — CHANGES_REQUESTED** [review]
Just one minor option name change. Good stuff.

File: README.md (hunk 1 above)
Comment (ValJed) [reply README.md]: Yup done !

File: lib/api.js (hunk 3 above)
Comment (ValJed) [reply lib/api.js]: Fixed

File: lib/api.js (hunk 2 above)
Comment (ValJed) [reply lib/api.js:164]: Modified

Comment (ValJed) [reply lib/api.js]: Fixed

**Review by boutell — APPROVED** [review]

**Pull request merged**